tests/routes/reviews.test.js

```diff
@@ -75,9 +75,7 @@ describe('TEST Suites for the REVIEW ACTIONS', () => {
         request(server)
           .post('/api/v1/businesses/0/reviews')
           .set({ authorization: token })
-          .send({
-            review: 'The food at your restaurant was heavenly',
-          })
+          .send(reviewSeed.setInput('The food at your restaurant was heavenly'))
```

Comment:
I think this is ok the way it was before, is there a reason for changing it?

Reply:
While I was fixing the errors the test after this one was throwing, I thought it could be because I didn't use .setInput since that's how I made input available in the reviewSeeder. But the errors was even caused by the code in the controller itself. I just left it like that since it's presence or a sense didn't seem to matter.